client/performance-profiler/components/newsletter-banner.tsx

```diff
@@ -19,36 +22,52 @@ const Heading = styled.div`
 	font-weight: 500;
 	line-height: 24px;
 	text-align: left;
+	color: var( --studio-white );
 `;
 
 const Body = styled.div`
-	color: var( --studio-gray-70 );
+	color: var( --studio-gray-20 );
```

Comment:
I would add `text-wrap: balance;` to avoid widow words

|Before | After|
|-------|------|
|![CleanShot 2024-09-04 at 16 05 06](https://github.com/user-attachments/assets/63d7d030-6280-4847-9547-909a4eb409fb)|![CleanShot 2024-09-04 at 16 05 22](https://github.com/user-attachments/assets/0aa49e15-0a2e-437a-a941-aae9f9b2db5c)|